src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> firstNode;
+    private Node<T> lastNode;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

за замовчуванням значення 0, можна не вказувати